content/en/containers/kubernetes/distributions.md

```diff
@@ -483,6 +484,75 @@ spec:
 More `values.yaml` examples can be found in the [Helm chart repository][1]
 More `DatadogAgent` examples can be found in the [Datadog Operator repository][2]
 
+## vSphere Tanzu Kubernetes Grid (TKG) {#TKG}
+
+TKG requires some small configuration changes, shown below. For example, tolerations are required to schedule the Node Agent on the `master` nodes.
+
+
+{{< tabs >}}
+{{% tab "Helm" %}}
+
+Custom `values.yaml`:
+
+```yaml
+datadog:
+  apiKey: <DATADOG_API_KEY>
+  appKey: <DATADOG_APP_KEY>
+  kubelet:
+    tlsVerify: false
+  # Enable the new `kubernetes_state_core` check.
+  kubeStateMetricsCore:
+    enabled: true
+# Add a toleration so that the agent can be scheduled on the control plane nodes.
+agents:
+  tolerations:
+    - key: node-role.kubernetes.io/master
+      effect: NoSchedule
+```
+
+{{% /tab %}}
+{{% tab "Operator" %}}
+
+DatadogAgent Kubernetes Resource:
+
+```yaml
+apiVersion: datadoghq.com/v1alpha1
+kind: DatadogAgent
+metadata:
+  name: datadog
+spec:
+  credentials:
+    apiSecret:
+      secretName: datadog-secret
+      keyName: api-key
+    appSecret:
+      secretName: datadog-secret
+      keyName: app-key
+  features:
+    # Enable the new `kubernetes_state_core` check.
+    kubeStateMetricsCore:
+      enabled: true
+  agent:
+    image:
+      name: "gcr.io/datadoghq/agent:latest"
+    config:
+      kubelet:
+        tlsVerify: false
```

Comment:
could you add a comment why `tlsVerify` need to be `false`.
Have you tried to mount a root certificat from the host?